composer.json

```diff
@@ -159,6 +164,7 @@
     "wpackagist-plugin/enhanced-media-library": "^2.3",
     "wpackagist-plugin/global-meta-box-order": "^1.0",
     "wpackagist-plugin/post-expirator": "^2.1",
+    "wpackagist-plugin/polylang": "^2.1",
```

Comment:
Is it good that Polylang is included, even if it's not needed in each project?

Reply:
True true, its just used in around 80-90% of projects :/